Riot/Modules/Room/RoomViewController.m

```diff
@@ -3433,7 +3455,12 @@ - (void)showAdditionalActionsMenuForEvent:(MXEvent*)selectedEvent inCell:(id<MXK
         // because it breaks everything
         if (selectedEvent.eventType != MXEventTypeRoomEncryption)
         {
-            [actionsMenu addAction:[UIAlertAction actionWithTitle:[VectorL10n roomEventActionRedact]
+            NSString *title = [VectorL10n roomEventActionRedact];
+            if (selectedEvent.eventType == MXEventTypePollStart) {
+                title = [VectorL10n roomEventActionRemovePoll];
+            }
```

Comment:
Could this be done without reassigning the new value if it's a poll?

Reply:
Sure, but I think that version of the code is uglier:
```
NSString *title;
if (selectedEvent.eventType == MXEventTypePollStart)
{
    title = [VectorL10n roomEventActionRemovePoll];
}
else
{
    title = [VectorL10n roomEventActionRedact];
}            

[actionsMenu addAction:[UIAlertAction actionWithTitle:title
```

Reply:
One solution would be adding an `isPoll` property to the event. Then you could

```objc
NSString *title = selectedEvent.isPoll ? VectorL10n.roomEventActionRemovePoll : VectorL10n.roomEventActionRedact
```

Reply:
I think I'll just go with the ugly version 😄